dashboard/app/models/contact_rollups_final.rb

```diff
@@ -15,4 +15,23 @@
 
 class ContactRollupsFinal < ApplicationRecord
   self.table_name = 'contact_rollups_final'
+
+  def self.overwrite_from_processed_table
+    truncate
+    insert_from_processed_table
+  end
+
+  def self.truncate
+    ActiveRecord::Base.connection.truncate('contact_rollups_final')
```

Comment:
nit: if this function is only used once it probably doesn't need to be its own function

Reply:
Done!